src/Util/Filter.php

```diff
@@ -17,18 +17,16 @@
  */
 final class Filter
 {
+    /** @var Blacklist */
+    private static $blacklist;
```

Comment:
What is your reasoning for using static attributes here?

Reply:
I choose to add the attributes to avoid add many params on the private methods that I added.

Reply:
But is there a specific reason why you declared these attributes `static`?

Reply:
It seems to me that we can not use non static attribut ion a static method.

https://3v4l.org/vqjCY

Reply:
Fair point, I blocked out the fact that `Filter` still uses static methods.

Reply:
I would prefer, though, not to have these attributes and that `$blacklist` and `$prefix` are passed as parameters to the private method. Thanks!

Reply:
ok I will do the change. 

Will you be interested to remove the static method from `Filter` in a future refactoring ?